obspy/imaging/waveform.py

```diff
@@ -1228,7 +1228,12 @@ def __sect_scale_traces(self):
 
     def __sect_init_time(self):
         """
-        Define the time vector for each trace
+        Define the time vector for each trace \n
```

Comment:
Can you remove the `\n` at the end but still at an extra newline after the first line? Also please end the line with a dot. I realize this was also wrong before ;-)